corputils/views.py

```diff
@@ -0,0 +1,141 @@
+from django.conf import settings
+from django.shortcuts import render_to_response
+from django.template import RequestContext
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.decorators import permission_required
+from django.shortcuts import HttpResponseRedirect
+
+from collections import namedtuple
+
+from authentication.managers import AuthServicesInfoManager
+from services.managers.eve_api_manager import EveApiManager
+from services.managers.evewho_manager import EveWhoManager
+from eveonline.models import EveCorporationInfo
+from eveonline.models import EveAllianceInfo
+from eveonline.models import EveCharacter
+from authentication.models import AuthServicesInfo
+from forms import CorputilsSearchForm
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+# Because corp-api only exist for the executor corp, this function will only be available in corporation mode.
+@login_required
+@permission_required('auth.corputils')
+def corp_member_view(request, corpid = settings.CORP_ID):
```

Comment:
in alliance mode, I've indicated `CORP_ID` is not required in settings.py - should I change that, or can you find another default? Perhaps if not `settings.IS_CORP` just render the corp search form.